lib/fs/fakefs.go

```diff
@@ -376,6 +411,10 @@ func (fs *fakefs) OpenFile(name string, flags int, mode FileMode) (File, error)
 		mtime: time.Now(),
 	}
 
+	if fs.insens {
+		base = UnicodeLowercase(base)
+	}
```

Comment:
The exists check is not taken into account

Reply:
Oh, thanks for pointing it out. There also appears to be a lovely race-condition in this function, so a thorough test is needed anyway. Will work on it.